telebot/types.py

```diff
@@ -8547,9 +8555,16 @@ def de_json(cls, json_string):
         elif obj['type'] == 'custom_emoji':
             del obj['type']
             return ReactionTypeCustomEmoji(**obj)
+        elif obj['type'] == 'paid':
+            del obj['type']
+            return ReactionTypePaid(**obj)
+        else:
+            raise ValueError(f"Unknown reaction type: {obj['type']}.")
 
     def __init__(self, type: str) -> None:
         self.type: str = type
+        if self.__class__ is ReactionType:
```

Comment:
same here: use type(self)

Reply:
<img width="653" height="220" alt="image" src="https://github.com/user-attachments/assets/fa4fbe5b-d16e-4bb1-8cfe-5d8bc050216b" />

We cannot! Thus "__ class __" is used here and for equality in other places...

Reply:
ok